cli/cmd/common.go

```diff
@@ -22,3 +23,15 @@ func handleCmdErr(cmd *cobra.Command, err error) {
 		log.Fatalf(err.Error())
 	}
 }
+
+// errNoConfig is returned if environment config file tt.yaml not found.
+var errNoConfig = errors.New("tt.yaml not found, you need to create tt" +
+	" environment config with 'tt init'")
+
+// checkConfig returns an error if environment config file tt.yaml not found.
+func checkConfig(cmdCtx *cmdcontext.CmdCtx) error {
```

Comment:
It is not clear from the function name what it does. Can we have something like "[is]ConfigFound" or "[is]ConfigFileLoaded" returning bool instead of error?

Reply:
I've renamed `checkConfig` -> `checkConfigExist`.

It looks easier:

```Go
if err != checkConfigExist(ctx); err != nil {
    return err
}
```

Than:

```Go
if ok != isConfigExist(ctx); !ok {
    return someErr
}
```

for me. But I can redo it if you insist.

Reply:
What do you think about 
```go
if !configExists(ctx) {
    . . . 
}
```
?
Errors are mostly used for exceptional cases. Missing config does not look exceptional for `exists` kind of functions.
But this is for sure exceptional for start command implementation, for example.